libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextHighlighting/TextHighlighting.styles.ts

```diff
@@ -28,6 +35,7 @@ export const textHighlightingStyles: () => IProcessedStyleSet<ITextHighlightingS
         {
           color: getPrimaryChartColor(theme),
           fontSize: theme.fonts.large.fontSize,
+          margin: "2px",
```

Comment:
When I added wrap to FluentUI Stack, it removed the margin for Labels but not Text, which squished the text with underlines together (but not the highlighted or plain text) - hence I needed to add the margin here.  I wonder if it might just be better to always use text and sometimes set fontweight instead of using Label, might be a good discussion point to bring up with Ke when he is back, since he preferred to use Label for bold words.